### PR TITLE
Document SQLite database

### LINKED ISSUE #2632: Document new sqlite database
PR: https://github.com/influxdata/influxdb/pull/21543

### PR DESCRIPTION
**Pull request by pierwill**
Closes #2632 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by pierwill** [comment]
Also added to CLI shortcode in base release branch here: https://github.com/influxdata/docs-v2/commit/ab9e24c06b372ff8e7cbda8ea9796668dc1f06ce

File: content/influxdb/v2.0/reference/config-options.md [hunk 1]
```diff
@@ -1469,13 +1470,63 @@ session-renew-disabled = true
...
+### sqlite-path
+
+Specify a user-defined path to the SQLite database file.
+
+| influxd flag    | Environment variable  | Configuration key |
+|:----------------|:----------------------|:------------------|
+| `--sqlite-path` | `INFLUXD_SQLITE_PATH` | `sqlite-path`     |
+
+**Default:** If not set explicitly, the default path for the SQLite database file
+is in the [`INFLUXD_BOLT-PATH`](#bolt-path) directory as `influxd.sqlite`.
```
Comment (sanderson) [inline content/influxdb/v2.0/reference/config-options.md:1483]:
```suggestion
Path to the SQLite database file.

**Default:** `~/.influxdbv2/influxd.sqlite`

| influxd flag    | Environment variable  | Configuration key |
|:----------------|:----------------------|:------------------|
| `--sqlite-path` | `INFLUXD_SQLITE_PATH` | `sqlite-path`     |
```

**Review by sanderson — CHANGES_REQUESTED** [review]
One suggestion to the added config option. Some other questions:

- What is SQLite used for? Is it an alternative to BoltDB or used in conjunction with BoltDB?
- If it is an alternative, does this add options to the `store` and `secret-store` options?

File: content/influxdb/v2.0/reference/config-options.md (hunk 1 above)
Comment (pierwill) [reply content/influxdb/v2.0/reference/config-options.md:1483]: See https://github.com/influxdata/influxdb/pull/21543/files#diff-51b7cb5da4f014baef4fe7492c9242fe230f76a48e3b798c28ae92ab4f21db61R292-R293 and resolved conversation there. It's strictly correct according to the code that the default for `sqlite-path` uses the same directory as `bolt-path`, which could be somewhere other than `~/.influxdbv2/influxd.sqlite`. I think it's better to be strict about this behavior and risk some confusion unless/until this code changes.

File: content/influxdb/v2.0/reference/config-options.md [hunk 2]
```diff
@@ -1469,13 +1470,64 @@ session-renew-disabled = true
...
+
+Specify a user-defined path to the SQLite database file.
+The SQLite database is used to store metadata for the notebook and annotations features.
+
+| influxd flag    | Environment variable  | Configuration key |
+|:----------------|:----------------------|:------------------|
+| `--sqlite-path` | `INFLUXD_SQLITE_PATH` | `sqlite-path`     |
+
+**Default:** If not set explicitly, the default path for the SQLite database file
+is in the [`INFLUXD_BOLT-PATH`](#bolt-path) directory as `influxd.sqlite`.
```
Comment (sanderson) [inline content/influxdb/v2.0/reference/config-options.md:1484]: The default value should be before the table. Simplified the default description.

```suggestion
### sqlite-path

Path to the SQLite database file.
The SQLite database is used to store metadata for notebooks and annotations.

**Default:** _`influxd.sqlite` in the same directory as the [bolt-path](#bolt-path)._

| influxd flag    | Environment variable  | Configuration key |
|:----------------|:----------------------|:------------------|
| `--sqlite-path` | `INFLUXD_SQLITE_PATH` | `sqlite-path`     |
```

File: content/influxdb/v2.0/reference/config-options.md [hunk 3]
```diff
@@ -1469,13 +1470,64 @@ session-renew-disabled = true
...
+
+###### influxd flag
+```sh
+influxd --sqlite-path
```
Comment (sanderson) [inline content/influxdb/v2.0/reference/config-options.md:1488]:
```suggestion
influxd --sqlite-path ~/.influxdbv2/influxd.sqlite
```

File: content/influxdb/v2.0/reference/config-options.md [hunk 4]
```diff
@@ -1469,13 +1470,64 @@ session-renew-disabled = true
...
+```
+
+###### Environment variable
+```sh
+export INFLUXD_SQLITE_PATH
```
Comment (sanderson) [inline content/influxdb/v2.0/reference/config-options.md:1493]:
```suggestion
export INFLUXD_SQLITE_PATH=~/.influxdbv2/influxd.sqlite
```

File: content/influxdb/v2.0/reference/config-options.md [hunk 5]
```diff
@@ -1469,13 +1470,64 @@ session-renew-disabled = true
...
+###### Configuration file
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[YAML](#)
+[TOML](#)
+[JSON](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```yml
+sqlite_path: <PATH>
```
Comment (sanderson) [inline content/influxdb/v2.0/reference/config-options.md:1505]:
```suggestion
sqlite_path: /users/user/.influxdbv2/influxd.sqlite
```

File: content/influxdb/v2.0/reference/config-options.md [hunk 6]
```diff
@@ -1469,13 +1470,64 @@ session-renew-disabled = true
...
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```toml
+sqlite_path = "<PATH>"
```
Comment (sanderson) [inline content/influxdb/v2.0/reference/config-options.md:1510]:
```suggestion
sqlite_path = "/users/user/.influxdbv2/influxd.sqlite"
```

File: content/influxdb/v2.0/reference/config-options.md [hunk 7]
```diff
@@ -1469,13 +1470,64 @@ session-renew-disabled = true
...
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```json
+{
+  "sqlite_path": "<PATH>"
```
Comment (sanderson) [inline content/influxdb/v2.0/reference/config-options.md:1516]:
```suggestion
  "sqlite_path": "/users/user/.influxdbv2/influxd.sqlite"
```

**Review by sanderson — CHANGES_REQUESTED** [review]
Some suggestions.

**Review by sanderson — APPROVED** [review]

**Pull request merged**